cuenca/resources/sessions.py

```
@@ -33,14 +40,15 @@ class Config:
             }
         }
+    )
 
     @classmethod
     def create(
         cls,
         user_id: str,
         type: SessionType,
-        success_url: Optional[str] = None,
-        failure_url: Optional[str] = None,
+        success_url=cast(Optional[AnyUrl], success_url),
```

Ruff (0.8.2)

Do not perform function call `cast` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)